server/cluster/fsm_test.go

```diff
@@ -0,0 +1,22 @@
+package cluster
+
+import (
+	"fmt"
+	"github.com/CeresDB/ceresdbproto/pkg/clusterpb"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestTransferLeader(t *testing.T) {
+
+	fsm := NewFSM(clusterpb.ShardRole_FOLLOWER)
+	fmt.Println(fsm.Current())
```

Comment:
`Println` is only used to debugging locally, and it is better to remove them when all things work.